#Stolen from kitcar-ros/KITcar_brain/src/messages/perception_msgs/msg/TrafficSigns.msg
Header header
TrafficSign[] sub_messages
